backends/bmv2/synthesizeValidField.cpp

```diff
@@ -37,7 +37,10 @@ class AddValidField final : public Modifier {
     bool preorder(IR::Type_Header* header) override {
         const cstring validField = V1ModelProperties::validField;
         if (header->getField(validField) != nullptr) return true;
-        auto field = new IR::StructField(validField, IR::Type::Bits::get(1));
+        auto annotations = new IR::Annotations();
+        annotations->add(new IR::Annotation(IR::Annotation::hiddenAnnotation,
+                                            IR::Vector<IR::Expression>()));
```

Comment:
The above 3 lines can be more compactly written as

    auto annotations = new IR::Annotations({new IR::Annotation(IR::Annotation::hiddenAnnotation, {})};

or even

    auto annotations = new IR::Annotations({new IR::Annotation("hidden", {})};